### ISSUE TITLE
Tedious or impossible for users of screen readers to find out if looping is on or off.

### ISSUE THREAD

**Issue by DavidBailes** [audacity/audacity#2019]
**Describe the bug**
The loop on/off option on the looping sub menu on the transport menu is not checked when looping is on.
Without this being checked it's either tedious or impossible for users of screen readers to know whether the looping setting is on. For users of Jaws and NVDA, they can go to the upper toolbars and check whether the loop button is pressed. But the Narrator screen reader does not read whether or not the button is pressed.
It should be easy for users of screen readers to check whether looping is on or not.

**To Reproduce**
1. Set looping to be on.
2. Open the looping sub menu on the transport menu
3. The loop on/off option is not checked

**Expected behavior**
If looping is on, then the loop on/off option should be checked. Compare with the options on the transport options sub menu, where options are checked.

**Additional information (please complete the following information):**
Tested on Windows 10.
